tests/testthat/test-webApp.R

```diff
@@ -0,0 +1,25 @@
+context("webApp")
+
+with_mock_crunch({
+    ds <- loadDataset("test ds")
+
+    test_that("Dataset URLs", {
+        with(temp.options(crunch.api="https://fake.crunch.io/api/v2/"), {
+            expect_identical(APIToWebURL(ds),
+                "https://fake.crunch.io/dataset/511a7c49778030653aab5963")
+        })
+        expect_identical(webToAPIURL("https://app.crunch.io/dataset/b6c2325a8de9438ebab5d9a42d376b90/browse/eyJhcHBTdGF0ZVN0b3JlIjp0cnVlLCJhbmFseXplIjp7fSwidmFyaWFibGVzTmF2aWdhdG9yIjp7Iml0ZW0iOiIvZWU2NTI0YWFjMzFiNDkyZjk4M2ZiYzM0MGJjODYzYzkvIn19"),
+            "https://app.crunch.io/api/datasets/b6c2325a8de9438ebab5d9a42d376b90/")
+        expect_identical(webToAPIURL("https://app.crunch.io/dataset/3f57d1924a914176b24969bc6cc9059d?variableId=000194"),
+            "https://app.crunch.io/api/datasets/3f57d1924a914176b24969bc6cc9059d/")
```

Comment:
more style than anything else: do we want to sanitize these ids? They are pretty long, and we could probably get away with shorter ones (the dataset ID identifier doesn't rely on the length of the ID, it doesn't appear). I'm also fine leaving them in since they do represent real world cases, and those ids don't give away anything...

Reply:
These tests were just copied over from their initial location to have all the webApp functions tested in one file. I can't really comment on whether we should modify them, but my intuition is that if they're already in the package we shouldn't change them unless there's a compelling reason.